src/picongpu/include/plugins/radiation/radFormFactor.hpp

```diff
@@ -82,6 +82,29 @@ namespace picongpu
   } // end namespace: radFormFactor_CIC_1Dy
 
 
+  namespace radFormFactor_Gauss_spherical
+  {
+    class radFormFactor
+    {
+    public:
+      HDINLINE radFormFactor(void)
+      { }
+
+      HDINLINE float_X operator()(const float_X N, const float_X omega, const vector_X observer_unit_vec) const
+      {
+
+        /* Form Factor for point-symmetric Gaussian charge distribution of N discrete electrons:
+         */
+        /* currently a fixes sigma of DELTA_T * c is used to describe the distribution - might become a parameter */
+        return sqrt(N + (N*N - N) * util::square(math::exp( -0.5 * util::square(omega * DELTA_T) ) ) ) ;
```

Comment:
Spaces in `N*N`, `float_X( 0.5 )` instead of plain `0.5` and spaces around argument in `util::square(omega * DELTA_T)`.